### DISCUSSION TITLE
ScreenShare: Connect subscription or just an account?

### DISCUSSION THREAD

**Discussion by physkets** [bordaigorl/rmview#138] [Q&A]
Do I need a paid subscription to one of the Connect plans in order to be able to use ScreenShare? Or is it enough if I have an account with the free plan?

**Comment by bordaigorl** [comment]
Unfortunately yes, they paywalled that feature.
Initially rmview shipped with a custom vnc server that did not require any authentication with remarkable, but then they changed the internals and engineering the vnc server got much harder.
See #103
